src/Pages/Home.jsx

```diff
@@ -0,0 +1,42 @@
+import { useEffect, useState } from "react"
+
+
+function Home(props) {
+    const competitions = props.competitions.map((competition, index) => {
```

Comment:
This is the way I'd recommend you do all of your maps going forward to keep things clean